app/services/EventReportService.scala

```diff
@@ -21,7 +21,7 @@ import com.google.inject.{Inject, Singleton}
 import connectors.EventReportConnector
 import connectors.cache.OverviewCacheConnector
 import models.ERVersion
-import models.enumeration.ApiType.{Api1826, Api1827, Api1832}
+import models.enumeration.ApiType.{Api1826, Api1827, Api1832, Api1830}
```

Comment:
I think all references to API 1832 can be removed. API 1830 is the POST, API 1832 is the GET for the same events and the enum lists show that they're completely equivalent. I think 1832 is redundant and so can be removed from the import here as well as in apiType, eventType (the val List) and any corresponding tests.